app/models/item_order.rb

```diff
@@ -2,6 +2,8 @@ class ItemOrder < ApplicationRecord
   belongs_to :item
   belongs_to :order
 
+  validates_presence_of :quantity
+
```

Comment:
Good idea to validate this. Maybe we want to validate presence of item and order too.